lib/wagyu/wasm/compiler.rb

```diff
@@ -107,6 +107,12 @@ def compile_instr(instr)
         new_var { @stack.pop(2).join(" != ") }
       when :eqz
         new_var { "#{@stack.pop} == 0" }
+      when :and
+        new_var { @stack.pop(2).join(" && ") }
+      when :or
+        new_var { @stack.pop(2).join(" || ") }
```

Comment:
Actually, "&&" and "||" have to be "&" and "|" respectively. 
https://medium.com/rubycademy/ruby-bitwise-operators-da57763fa368

Reply:
@edvakf fixed! d8bb610